### DISCUSSION TITLE
Pushing object with TurtleBot4

### DISCUSSION THREAD

**Discussion by paoloelle** [turtlebot/turtlebot4#471] [General]
Hi all, I'm developing a project where I'm planning to use a TurtleBot4 robot to push some objects. I was thinking of using balls as objects and wanted to know the maximum weight a TurtleBot4 can push. 
I'd appreciate any suggestion you can give. Thanks!

**Comment by slowrunner** [comment]
(Don't know if you are still working on this, and perhaps solved it on your own?)

This question is not easily answered because the value depends on the friction of the object with the floor, which will vary with the texture of the flooring and the materials involved.

What I can offer is a measure of how much force my Turtlebot4 (on my rough and perhaps slightly dusty ceramic tile floor) can apply to an object before the wheels started to slip.  That value is about 2000 grams.

Using an approximate coefficient of friction for a cardboard box on a smooth floor of 0.25, the Turtlebot4 might be able to push the box filled with less than 8kg.  Using a safety factor of 2 for all the guessing, the Turtlebot4 should be able to push a cardboard box filled with 3-4 kgs.

It is possible you will need to turn off the bumper reaction, unless you approach the objects slow enough not to trigger the bumper.

Did you make any video of pushing your objects?  Sounds like a great project.
